src/os/bluestore/ceph-aio.h

```diff
@@ -2,7 +2,15 @@
 // vim: ts=8 sw=2 smarttab
 
 #pragma once
-# include <libaio.h>
+
```

Comment:
might want to revise the topic of this commit to s/build/os/

Reply:
@wjwithagen this comment is not addressed.

and apparently this change is not merely renaming this file. please split this change into two commit:

1. rename this file
1. add the POSIX AIO bits to `ceph-aio.h`. 

and squash the POSIX AIO change into the bluestore enabling commit.